.cicd/run.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -eo pipefail
+. ./.cicd/helpers/general.sh
+. ./$HELPERS_DIR/execute.sh
+
+execute mkdir -p $ROOT_DIR/build
+
+if [[ $(uname) == Darwin ]]; then
+    MAC_CMAKE="cmake -DCMAKE_BUILD_TYPE=Release .."
+    MAC_MAKE="make -j$JOBS"
+    MAC_TEST="ctest -j$JOBS --output-on-failure -T Test"
+    cd $ROOT_DIR
+    ccache -s
+    cd build
+    if [[ $BUILDKITE ]]; then
+        if [[ $ENABLE_BUILD ]]; then
+            execute $MAC_CMAKE
+            execute $MAC_MAKE
+        elif [[ $ENABLE_TEST ]]; then
+            execute $MAC_TEST
+        fi
+    elif [[ $TRAVIS ]]; then
+        execute mkdir -p wasms
+        execute $MAC_CMAKE
+        execute $MAC_MAKE
+        #execute $MAC_TEST
+    fi
+
+else # Linux
+
+    . ./$HELPERS_DIR/docker.sh
+    . ./$HELPERS_DIR/docker-hash.sh
+
+    BUILD_COMMANDS="cd /workdir/build && cmake -DCMAKE_TOOLCHAIN_FILE=/workdir/.cicd/helpers/clang.make -DENABLE_TESTS=ON .. && make -j$JOBS"
+    TEST_COMMANDS="cd /workdir/build && ctest -j$JOBS --output-on-failure -T Test"
+
+    # Docker Run Arguments
+    ARGS=${ARGS:-"--rm -v $(pwd):/workdir"}
```

Comment:
Variables are for things which are used in many places, or which we expect to change frequently. Let's hard-code this.